src/Maker/MakeResetPassword.php

```diff
@@ -249,7 +249,7 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
                 'email_getter' => $this->emailGetterMethodName,
                 'email_field' => $this->emailPropertyName,
                 'password_class_details' => ($passwordClassDetails = $generator->createClassNameDetails($passwordHasher, '\\')),
-                'password_variable_name' => sprintf('$%s', lcfirst($passwordClassDetails->getShortName())), // @legacy see passwordHasher conditional above
+                'password_variable_name' => str_replace('Interface', '', sprintf('$%s', lcfirst($passwordClassDetails->getShortName()))), // @legacy see passwordHasher conditional above
```

Comment:
```suggestion
                'password_hasher_variable_name' => str_replace('Interface', '', sprintf('$%s', lcfirst($passwordClassDetails->getShortName()))), // @legacy see passwordHasher conditional above
```
Same for password_class_details ?